questionnaire-ja.json

```diff
@@ -1,434 +1,434 @@
-          "number": "1.b",
-          "type": "YES_NO",
-          "specReference": [
-            "1.1",
-            "1.1.2"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Have you identified the roles and responsibilities that affect the performance and effectiveness of the Program?",
-          "number": "1.c",
-          "type": "YES_NO",
-          "specReference": [
-            "1.2",
-            "1.2.1"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Have you identified and documented the competencies required for each role?",
-          "number": "1.d",
-          "type": "YES_NO",
-          "specReference": [
-            "1.2",
-            "1.2.2"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Have you documented the assessed competence for each Program participant?",
-          "number": "1.e",
-          "type": "YES_NO",
-          "specReference": [
-            "1.2",
-            "1.2.3"
-          ]
-        },
-        {
-          "question": "Have you documented the awareness of your Program participants on the following topics?",
-          "number": "1.f",
-          "type": "SUBQUESTIONS",
-          "specReference": [
-            "1.3",
-            "1.3.1"
-          ],
-          "minNumberValidatedAnswers": 4,
-          "subQuestions": {
-            "1.f.i": {
-              "correctAnswer": "Yes",
-              "question": "The open source policy and where to find it;",
-              "number": "1.f.i",
-              "type": "YES_NO",
-              "specReference": [
-                "1.3"
-              ]
-            },
-            "1.f.ii": {
-              "correctAnswer": "Yes",
-              "question": "Relevant open source objectives;",
-              "number": "1.f.ii",
-              "type": "YES_NO",
-              "specReference": [
-                "1.3"
-              ]
-            },
-            "1.f.iii": {
-              "correctAnswer": "Yes",
-              "question": "Contributions expected to ensure the effectiveness of the Program;",
-              "number": "1.f.iii",
-              "type": "YES_NO",
-              "specReference": [
-                "1.3"
-              ]
-            },
-            "1.f.iv": {
-              "correctAnswer": "Yes",
-              "question": "The implications of failing to follow the Program requirements.",
-              "number": "1.f.iv",
-              "type": "YES_NO",
-              "specReference": [
-                "1.3"
-              ]
-            }
-          }
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a process for determining the scope of your Program?",
-          "number": "1.g",
-          "type": "YES_NO",
+            "1.1",
+            "1.1.2"
+          ]
+        },
+        {
+          "correctAnswer": "はい",

```

Comment:
The correctAnswer should not be translated

Reply:
Done locally. Pushing later.